.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
+      "pattern": ["**", "!.git/**", "!node_modules/**"]
+    }
+  },
```

Comment:
- [ ] [REQUIRED] This pull request(PR) can be improved by updating the title and description.
Pull Request should have a descriptive title telling what changes are inside.
Pull Request should have a summary describing introduced changes in more detail.
e.g Title -> Hello Microverse project and the summary -> Mention the things you did in brief detail.

See the image below for an example of a descriptive PR summary 👇 
![](https://user-images.githubusercontent.com/93039359/191243176-550fc938-97e4-4cf7-969c-ce112fd46536.png)